src/cascadia/UnitTests_TerminalCore/ScreenSizeLimitsTest.cpp

```diff
@@ -22,7 +22,8 @@ namespace TerminalCoreUnitTests
         MockTermSettings(int32_t historySize, int32_t initialRows, int32_t initialCols) :
             _historySize(historySize),
             _initialRows(initialRows),
-            _initialCols(initialCols)
+            _initialCols(initialCols),
+            _tabTitle(winrt::to_hstring(""))

```

Comment:
because you've removed TabTitle from the `idl` you no longer need to have it here in MockTermSettings :smile: